schema_source/jetstream/api/v1/definitions.json

```diff
@@ -5,6 +5,33 @@
   "description": "Shared definitions for the JetStream API",
   "type": "object",
   "definitions": {
+    "golang_duration_nanos": {
+      "$comment": "nanoseconds depicting a duration in time",
+      "$ref": "#/definitions/golang_int64"
+    },
+    "golang_int": {
+      "$comment": "integer with a dynamic bit size depending on the platform the cluster runs on, can be up to 64bit",
+      "minimum": -9223372036854775808,
+      "maximum": 9223372036854775807
```

Comment:
@kozlovic mainly added you to review to double check this is a correct scription of `int`

Reply:
Yes, it is the correct range for `int` on a 64bit platform.